AprilTagsVision/vision.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
```

Comment:
I don't have time to test this at the moment, but if you want to try it on your computer then just install `robotpy[cscore]` and `opencv-python` and you can run this file locally with your computer's camera via `python -m cscore vision.py`.

Probably should add a comment about that here too.

Reply:
Done!